quines/README.md

```diff
@@ -0,0 +1,11 @@
+# Binaris Quine Collection
+
+A [quine](https://en.wikipedia.org/wiki/Quine_(computing)) is a non-empty computer program which takes no input and produces a copy of its own source code as its only output.  
+An [ouroboros](https://en.wikipedia.org/wiki/Ouroboros) is a serpent eating its own tail, and a recursion of the quine concept.
```

Comment:
Not sure I like the wording of 

> and a recursion of the quine concept.

Reply:
it's based on https://en.wikipedia.org/wiki/Quine_(computing)#Ouroboros_programs
you're the native english speaker. suggest something better :-)

Reply:
1. Even if Wiki is correct most of the time it doesn't necessarily mean it's digestable most of the time.
2. The first thing I did was copy your sentence and then visit the Wiki using 'c-f'. "recurs" is only found twice and definitely not with your wording.

My suggestion would be.

"is a serpent eating its own tail, and can be thought of as a 'recursive' quine."